### ISSUE TITLE
HW acceleration and Immediate processing mode

### ISSUE THREAD

**Issue by skelletR** [nikita36078/J2ME-Loader#424]
**Emulator version:** 1.3.7-play 

**Game version:**

**Game resolution:**
``(For example, 240x320 or 640x360)``

**Device:**
``(Sony Xperia Z5 )``

**Android version:**7.1.1

**Description of the issue:**
``(I have a question if I enable Immediate processing mode and HW acceleration.. The only damage if can is in the games inside in Emulator not in my phone system something like that etc?? )``

**Comment by nikita36078** [comment]
Yes, it can cause crashes in the emulator, but it's completely safe for system and phone.

**Comment by skelletR** [comment]
Oh.. That is  nice to hear for phone.. And I hope in future you're Emulator is full of stable.. And now is the best really
